Network/dedicated-load-balancer-basic-management.md

```diff
@@ -12,12 +12,16 @@
 <ul>
   <li>Must have a dedicated Netscaler available in your environment</li>
   <li>Must have an Admin login to your netscaler</li>
-  <li>Must have Java installed and configured correctly. See KB:&nbsp;https://t3n.zendesk.com/entries/55258750-How-to-configure-Java-settings-to-access-web-user-interfaces
-    <a href="https://t3n.zendesk.com/entries/27208970-How-to-access-the-web-interface-of-a-Netscaler-Load-Balancer">
+  <li>Must have Java installed and configured correctly. See KB:&nbsp;how-to-configure-java-settings
+    <a href="https://www.ctl.io/knowledge-base/general/how-to-configure-java-settings-to-access-web-user-interfaces/">
```

Comment:
@aaronsr put the link text `how-to-configure-java-settings` inside the `<a>` tag